pkg/plugin/validate.go

```diff
@@ -0,0 +1,20 @@
+// Package plugin holds the implementation of a plugin.
+package plugin
+
+import "fmt"
+
+func (p *plugin) validateConfig() (errs []error) {
+	if p.config.SyncImage == "" {
```

Comment:
This is mandatory in RP side, but not in our code base. We should swap maybe?

Reply:
agree